src/sns-lambda/index.ts

```diff
@@ -10,7 +10,7 @@ const buildNotificationMessage = (sns: SNSMessage): string =>
     'You have a new notification:',
     sns.Timestamp,
     '',
-    `*${sns.Subject}*`,
+    `*${sns.Subject ?? ''}*`,
```

Comment:
This is the only line I’m unclear on- is it building a notification message, but having an empty string if there’s no sns.Subject in the message

Reply:
Yeah - for some reason `sns.Subject` has type `string | unknown` which [`@typescript-eslint`](https://typescript-eslint.io/rules/restrict-template-expressions) doesn't like now.

I've just decided to slap a `?? ''` because I've not got the detailed types in front of me and ultimately this is an example lambda for using SNS so this actual code is expected to be modified anyway 🤷

Reply:
Ah sweet, that makes sense 😊